gooddata-sdk/tests/catalog/test_catalog_workspace_content.py

```diff
@@ -0,0 +1,215 @@
+def test_load_and_put_declarative_analytics_model(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    path = _current_dir / "load" / "declarative_analytics_model" / "declarative_analytics_model_demo.yaml"
+    identifier = test_load_and_put_declarative_analytics_model.__name__
+    _set_up_workspace_ldm(sdk, test_config["workspace"], identifier)
+    analytics_model_e = sdk.catalog_workspace_content.get_declarative_analytics_model(test_config["workspace"])
+
+    try:
+        sdk.catalog_workspace_content.load_and_put_declarative_analytics_model(identifier, path)
+        analytics_model_o = sdk.catalog_workspace_content.get_declarative_analytics_model(identifier)
+        assert analytics_model_e == analytics_model_o
+        # assert analytics_model_e.to_api().to_dict() == analytics_model_o.to_api().to_dict()
+    finally:
+        sdk.catalog_workspace.delete_workspace(identifier)
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "demo_put_declarative_analytics_model.json"))
+def test_put_declarative_analytics_model(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    identifier = test_put_declarative_analytics_model.__name__
+    _set_up_workspace_ldm(sdk, test_config["workspace"], identifier)
+    analytics_model_e = sdk.catalog_workspace_content.get_declarative_analytics_model(identifier)
+
+    try:
+        sdk.catalog_workspace_content.put_declarative_analytics_model(identifier, analytics_model_e)
+        analytics_model_o = sdk.catalog_workspace_content.get_declarative_analytics_model(identifier)
+        assert analytics_model_e == analytics_model_o
+        assert analytics_model_e.to_api().to_dict() == analytics_model_o.to_api().to_dict()
+    finally:
+        sdk.catalog_workspace.delete_workspace(identifier)
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "demo_put_declarative_ldm.json"))
+def test_put_declarative_ldm(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    identifier = test_put_declarative_ldm.__name__
+    workspace = CatalogWorkspace(workspace_id=identifier, name=identifier)
+    sdk.catalog_workspace.create_or_update(workspace)
+
+    ldm_e = sdk.catalog_workspace_content.get_declarative_ldm(test_config["workspace"])
+    try:
+        sdk.catalog_workspace_content.put_declarative_ldm(identifier, ldm_e)
+        ldm_o = sdk.catalog_workspace_content.get_declarative_ldm(identifier)
+        assert ldm_e == ldm_o
+        assert ldm_e.to_api().to_dict() == ldm_o.to_api().to_dict()
+    finally:
+        sdk.catalog_workspace.delete_workspace(identifier)
+
+
+@gd_vcr.use_cassette(str(_fixtures_dir / "demo_get_declarative_analytics_model.json"))
+def test_get_declarative_analytics_model(test_config):
+    sdk = GoodDataSdk.create(host_=test_config["host"], token_=test_config["token"])
+    path = _current_dir / "expected" / "declarative_analytics_model.json"
```

Comment:
I would store these json files into "fixtures/workspace_content", not into "expected".

Reply:
These files represent expected content. Do we really want to mix them with fixtures? There is also declarative_workspaces.json, I assume that this file should be part of the fixtures/workspaces right?

Reply:
I get it, let's keep it like it is